model/event/subscriptions.go

```diff
@@ -38,28 +36,73 @@ type Subscription struct {
 	Selectors      []Selector    `bson:"selectors,omitempty"`
 	RegexSelectors []Selector    `bson:"regex_selectors,omitempty"`
 	Subscriber     Subscriber    `bson:"subscriber"`
+	ExtraData      interface{}   `bson:"extra_data,omitempty"`
+}
+
+type unmarshalSubscription struct {
+	ID             bson.ObjectId `bson:"_id"`
```

Comment:
do we need to have this be an objectID as an object id?

Reply:
Have to, no. Changing this involves changing a bunch of structs. Is there any reason you prefer this to `bson.ObjectId`s?

Reply:
because we're going to switch bson libraries soon. :)